tests/js/client/shell/shell-api-database.js

```diff
@@ -174,6 +174,12 @@ function dealing_with_database_manipulation_methodsSuite () {
 
     tearDown: function() {
       arango.DELETE(api + `/${name}`);
+      db._flushCache();
+      db._users.toArray().forEach(user => {
+        if (user.user !== "root") {
+          arango.DELETE_RAW("/_api/user/" + user.user);
```

Comment:
Not sure what's in the user names here, but URL-encoding them sounds like a good idea in general.
```suggestion
          arango.DELETE_RAW("/_api/user/" + encodeURIComponent(user.user));
```